src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -183,10 +178,12 @@ const std::shared_ptr<AbstractOperator>& SQLPipelineStatement::get_physical_plan
     }
   }
 
+  bool pqp_cacheable = true;
```

Comment:
do you need this variable? `get_optimized_logical_plan()` sets the `_optimized_logical_plan` member, which contains the info you need later.